reporter/src/funTest/assets/static-html-reporter-test-expected-output.html

```diff
@@ -618,20 +618,30 @@ <h3 class="ort-excluded">Packages</h3>
             <td><a href="#Gradle:org.ossreviewtoolkit:nested-fake-project:1.0.0-pkg-1">1</a></td><td>Gradle:org.ossreviewtoolkit:nested-fake-project:1.0.0</td><td></td><td><em>Declared Licenses:</em>
```

Comment:
Commit message nit: In the first line of the body, there is a superfluous "the" after "enhanced".